common/beerocks/bwl/include/bwl/ap_wlan_hal.h

```diff
@@ -53,6 +53,7 @@ class ap_wlan_hal : public virtual base_wlan_hal {
         ACS_Failed,
 
         CSA_Finished,
+        CTRL_EventChannelSwitch,
```

Comment:
I think we can drop the prefix like other events anf call this `ChannelSwitch`

Reply:
Done